### ISSUE TITLE
MLLM Support

### ISSUE THREAD

**Issue by Ryan-ZL-Lin** [joaomdmoura/crewAI#166]
Hi
First of all, thanks for your amazing work here.
I’m wondering whether CrewAI would support any MLLM (multimodal large language model)? Since it’s more suitable in my use case. For example, the agent can take a image from somewhere and invoke the API from any MLLM to understand the meaning of this image.

**Comment by matsuobasho** [comment]
I'm a new user who's experimented with CrewAI a little and haven't tried it but why not?  If you tell it what image to ingest and specify the multimodal model to use then I don't see why not.  Try it and report back what you see if you have questions, would be curious to know, too.

**Comment by joaomdmoura** [comment]
Yes! we are def going multimodal, I've tested yet but it could work already, that said I want to have a better DSL around it. Marking this as feature accepted

**Comment by Ryan-ZL-Lin** [comment]
> Yes! we are def going multimodal, I've tested yet but it could work already, that said I want to have a better DSL around it. Marking this as feature accepted

Thanks @joaomdmoura
look forward seeing this feature released soon.
